controllers/team.controller.js

```diff
@@ -7,6 +7,30 @@ const Services = {
 };
 const Util = require("../middlewares/util.middleware");
 
+/**
+ * @async
+ * @function findById
+ * @param {*} req 
+ * @param {*} res 
+ * @return {JSON} Success or error status
+ * @description Finds a team by it's mongoId that's specified in req.param.id in route parameters.
+ */
+async function findById(req, res) {
+    const team = await Services.Team.findById(req.params.id);
+
+    if (team) {
+        return res.status(200).json({
+            message: "Successfully retrieved team information",
+            data: team.toJSON()
+        });
+    } else {
+        return res.status(400).json({
```

Comment:
This should be a 404, since team does not exist. Message should be something like "Team does not exist" or something